### ISSUE TITLE
Add badges

### ISSUE THREAD

**Issue by eyablokov** [kelseyhightower/nocode#1815]
Please, add GitHub badges (like, build passing, code coverage, ...).

**Comment by lhcgoncalves** [comment]
Duplicate of #

**Comment by RustyRaptor** [comment]
No, badges are a security concern. We are planning to purge the markdown in the README completely.
